commitrmnocb/types.go

```diff
@@ -35,12 +37,16 @@ func NewCommitQuery(rmnOnRampMaxSeqNums []plugintypes.SeqNumChain, merkleRoots [
 }
 
 type Observation struct {
-	MerkleRoots        []cciptypes.MerkleRootChain     `json:"merkleRoots"`
-	GasPrices          []cciptypes.GasPriceChain       `json:"gasPrices"`
-	TokenPrices        []cciptypes.TokenPrice          `json:"tokenPrices"`
-	OnRampMaxSeqNums   []plugintypes.SeqNumChain       `json:"onRampMaxSeqNums"`
-	OffRampNextSeqNums []plugintypes.SeqNumChain       `json:"offRampNextSeqNums"`
-	FChain             map[cciptypes.ChainSelector]int `json:"fChain"`
+	MerkleRoots []cciptypes.MerkleRootChain `json:"merkleRoots"`
+	GasPrices   []cciptypes.GasPriceChain   `json:"gasPrices"`
+	// Prices for tokens from the feed on the feed chain
+	FeedTokenPrices []cciptypes.TokenPrice `json:"feedTokenPrices"`
+	// Prices for tokens from the PriceRegistry on the dest chain
+	PriceRegistryTokenUpdates map[types.Account]sharedtypes.NumericalUpdate `json:"priceRegistryTokenPrices"`
```

Comment:
Thinking about ditching the `NumericalUpdate` and have it TokenPrice without the timestamp info. We already rely on observation timestamps and no need for the last update info. WDYT? @makramkd